internal/pkg/cache/cache.go

```diff
@@ -182,7 +182,7 @@ func (h *Handle) CleanCache(cacheType string, dryRun bool, days int) (err error)
 
 		sylog.Infof("Removing %s cache entry: %s", cacheType, f.Name())
 		if !dryRun {
-			// We RemoveAll in case the entry is a directory from Apptainer <3.6
+			// We RemoveAll in case the entry is a directory from Apptainer < v1.0.0
```

Comment:
This should be Singularity <3.6

Reply:
++

Reply:
both are correct

Reply:
done

Reply:
if !dryRun {
-			// We RemoveAll in case the entry is a directory from Apptainer < v1.0.0
+			// We RemoveAll in case the entry is a directory from Singularity (prior to 3.6)
 			err := os.RemoveAll(path.Join(dir, f.Name()))

Reply:
> both are correct

No, because there was no Apptainer before 1.0.0 and because Singularity >= 3.6 doesn't have the problem.